docs/quickstart/docker-compose.yml

```diff
@@ -53,7 +53,7 @@ services:
 
   # Runs the Kafka KSQL data generator for topic called "pageviews"
   ksql-datagen-pageviews:
-    image: "confluentinc/ksql-examples:0.2"
+    image: "confluentinc/ksql-examples:0.3-rc1"
```

Comment:
you probably don't want to push these changes just yet.

Reply:
Yes, removed them.